core/src/test/resources/org/elasticsearch/index/analysis/test1.json

```diff
@@ -17,10 +17,6 @@
                 },
                 "my":{
                     "type":"myfilter"
-                },
```

Comment:
Naming this the same as the one for compound analysis seems bad. Maybe each one should have its own name.